src/Form/Template/LegacyFormSettingCompatibility.php

```diff
@@ -93,4 +94,41 @@ public function save( $formId, $settings ) {
 			}
 		}
 	}
+
+	/**
+	 * Migrate existing legacy form settings.
+	 *
+	 * @param  int  $formId
+	 *
+	 * @since 2.7.0
+	 */
+	public static function migrateExistingFormSettings( $formId ) {
+		if ( ! Utils::isLegacyForm( $formId ) || 'edit' !== give_clean( $_GET['action'] ) ) {
+			return;
+		}
+
+		if ( Give()->form_meta->get_meta( $formId, '_give_legacy_form_template_settings', true ) ) {
+			return;
+		}
+
+		$mapToSetting = [
```

Comment:
@ravinderk you double and triple checked these are all the options?

Reply:
@DevinWalker These are the only options (form style-related ) that move to the legacy form template.
Total eight settings we moved: https://github.com/impress-org/givewp/blob/8afe9bd7511cf733fe3752a620eb9377b8a0bbac/src/Views/Form/Templates/Legacy/optionConfig.php#L11